src/client/components/app/Transactions/RevokeConsentTx.tsx

```diff
@@ -21,70 +25,42 @@ const StyledAmountInput = styled(TxTTLInput)``;
 interface RevokeConsentProps {
   header: string;
   onClose: () => void;
-  onSelectedCreditLineChange: Function;
-  onPositionChange: (data: { credit?: string; amount?: string }) => void;
 }
 
 export const RevokeConsentTx: FC<RevokeConsentProps> = (props) => {
   const { t } = useAppTranslation('common');
   const dispatch = useAppDispatch();
-  const { header, onClose, onPositionChange } = props;
+  const { header, onClose } = props;
   const [transactionCompleted, setTransactionCompleted] = useState(0);
   const [transactionLoading, setLoading] = useState(false);
   const [targetAmount, setTargetAmount] = useState('1');
   const [errors, setErrors] = useState<string[]>(['']);
   const selectedCredit = useAppSelector(LinesSelectors.selectSelectedLine);
   const selectedPosition = useAppSelector(LinesSelectors.selectSelectedPosition);
+  const selectedProposal = useAppSelector(LinesSelectors.selectSelectedProposal);
   const walletNetwork = useAppSelector(WalletSelectors.selectWalletNetwork);
-  const setSelectedCredit = (lineAddress: string) => dispatch(LinesActions.setSelectedLineAddress({ lineAddress }));
+  const walletAddress = useAppSelector(WalletSelectors.selectSelectedAddress);
   const positions = useAppSelector(LinesSelectors.selectPositionsForSelectedLine);
 
-  //Calculate maximum withdraw amount, then humanize for readability
-  const getMaxWithdraw = () => {
-    if (!selectedPosition) {
-      setErrors([...errors, 'no selected position']);
-      return '0';
+  // state for proposal params
+  const [targetTokenAmount, setTargetTokenAmount] = useState('0');
+  const [drate, setDrate] = useState('');
```

Comment:
dont think we need these local variables because we dont update anything. Can just read directly from position data

Reply:
TODO - same/similar logic is being used in `AddCollateralTx`, `AddCreditPositionTx`, `ReleaseCollateralTx`, and now `RevokeConsentTx` as Mo and I built off what was available in `AddCreditPositionTx` and `AddCollateralTx`. This is a refactor that should be applied across those components.

Reply:
okay fixed this for `RevokeConsentTx`